gen3-client/g3cmd/utils.go

```diff
@@ -169,15 +181,20 @@ func GenerateMultipartPresignedURL(g3 Gen3Interface, key string, uploadID string
 }
 
 // CompleteMultipartUpload helps sending requests to FENCE to complete a multipart upload
-func CompleteMultipartUpload(g3 Gen3Interface, key string, uploadID string, parts []MultipartPartObject) error {
+func CompleteMultipartUpload(g3 Gen3Interface, key string, uploadID string, parts []MultipartPartObject, bucketName string) error {
 	request := new(jwt.Request)
 	configure := new(jwt.Configure)
 	function := new(jwt.Functions)
 
 	function.Config = configure
 	function.Request = request
 
-	multipartCompleteObject := MultipartCompleteRequestObject{Key: key, UploadID: uploadID, Parts: parts}
+	var bucketPtr *string
```

Comment:
I think if you just passed through the bucketName as a string you wouldn't have to do all this every time